lua/wikis/commons/infobox/infobox_unofficial_world_champion.lua

```diff
@@ -148,6 +159,14 @@ function UnofficialWorldChampion:createInfobox()
 	return self:build(widgets)
 end
 
+---@param key string
+---@return standardOpponent
+function UnofficialWorldChampion:_parseOpponentArg(key)
+	return Opponent.readOpponentArgs(
+		Json.parseIfString(self.args[key]) or Opponent.tbd()
```

Comment:
Are we sure we want to display TBD if for instance a wiki chooses not to track that metric? Or does all wikis track all metrics?